src/main/java/com/exadel/sandbox/employee/service/impl/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package com.exadel.sandbox.employee.service.impl;
+
+import com.exadel.sandbox.employee.service.EmployeeService;
+import com.exadel.sandbox.exception.EntityNotFoundException;
+import com.exadel.sandbox.employee.entity.Employee;
+import com.exadel.sandbox.employee.dto.EmployeeDto;
+import com.exadel.sandbox.employee.repository.EmployeeRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Component
+public class EmployeeServiceImpl implements EmployeeService {
+
+    @Autowired
+    EmployeeRepository employeeRepository;
```

Comment:
I think you should add: private final

Reply:
If we use @Autowired just private is enough, but if we have @RequiredArgsConstructor instead of @Autowired then we should add private final